services/graph/pkg/service/v0/drives.go

```diff
@@ -408,7 +409,21 @@ func (g Graph) UpdateDrive(w http.ResponseWriter, r *http.Request) {
 
 	if drive.Quota.HasTotal() {
 		user := revactx.ContextMustGetUser(r.Context())
-		canSetSpaceQuota, err := g.canSetSpaceQuota(r.Context(), user)
+
+		// NOTE: a space admin cannot get a space by ID. We need to fetch all spaces and search for it
```

Comment:
😞 Can you open a bug about that ...

Reply:
https://github.com/owncloud/ocis/issues/5867